src/uct/ib/mlx5/ib_mlx5_dv.c

```diff
@@ -0,0 +1,30 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2001-2018.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#include "ib_mlx5.h"
+#include "ib_mlx5_log.h"
+#include <ucs/arch/bitops.h>
+
+#if HAVE_DECL_MLX5DV_INIT_OBJ
+ucs_status_t uct_ib_mlx5dv_init_obj(uct_ib_mlx5dv_t *obj, uint64_t type)
+{
+    int ret;
+
+    ret = mlx5dv_init_obj(&obj->dv, type);
+    if (ret != 0) {
+        ucs_error("DV failed to get mlx5 information. Type %lx. "
+                  "Please make sure the installed libmlx5 version matches "
```

Comment:
actually, we can remove the message "Please make sure the installed libmlx5 version matches ..." because now we don't access internal data structures